scoreboard/app.py

```diff
@@ -66,6 +67,10 @@ def create_app(config=None):
         app.logger.handlers[0].setFormatter(log_formatter)
         logging.getLogger().handlers[0].setFormatter(log_formatter)
 
+    #Scss is expecting this
+    app.debug = app.config.get('DEBUG')
+    scss_compiler = Scss(app, static_dir='static/css', asset_dir='static/scss')
+    scss_compiler.update_scss()
```

Comment:
What specifically won't work. The debug portion, or the entire compilation process?